common/option-model.cpp

```diff
@@ -432,7 +432,14 @@ bool option_model::draw_slider( notifications_model & model,
                 }
                 else
                 {
-                    set_option( opt, new_value, error_message );
+                    // run when the value is valid and the enter key is pressed to submit the new value
+                    auto option_was_set = set_option(opt, new_value, error_message);
+                    if (option_was_set)
+                    {
+                        if (invalidate_flag)
+                            *invalidate_flag = true;
```

Comment:
Is this the step that will make it refresh?

Reply:
yes, setting the invalidate flag to true seems to make it refresh